src/Sulu/Component/Rest/ListBuilder/Doctrine/FieldDescriptor/DoctrineGroupConcatFieldDescriptor.php

```diff
@@ -61,9 +71,15 @@ public function getSelect()
     }
 
     /**
-     * Returns all the joins required for this field.
-     *
-     * @return DoctrineJoinDescriptor[]
+     * {@inheritdoc}
+     */
+    public function getSimpleSelect()
```

Comment:
hmm maybe call it `getWhere` because its used to generate where statement. should i change it?